livestyled/models/form_field.py

```diff
@@ -0,0 +1,110 @@
+from typing import Dict, List
+
+
+class FormField:
+    def __init__(
+            self,
+            id,
+            type=None,
+            key=None,
+            required=None,
+            translations=None,
+            sort_id=None,
+            validation_regex=None,
+            select_options=None,
+            auto_fill=None,
+    ):
+        self._id = id
+        self.type = type
+        self.key = key
+        self.validation_regex = validation_regex
+        self.required = required
+        self.select_options = select_options
+        self.sort_id = sort_id
+        self.auto_fill = auto_fill
+        if translations:
+            self.fields = [FormFieldTranslation(**fft) for fft in translations]
```

Comment:
`fft -> translation`. `translation` makes a better name than `fft`

```
self.fields = translations and [FormFieldTranslation(**translation) for translation in translations]
```

The above will assign None if None else it will generate the list and we eliminate the if statement.